tests/conftest.py

```diff
@@ -196,6 +205,20 @@ def pytest_runtest_setup(item: "pytest.Item"):
         return
     if any(True for mark in item.iter_markers(name="slowtest")):
         pytest.skip("Skipping slow tests")
+    if any(True for mark in item.iter_markers(name="db_migration_test")):
```

Comment:
I just realize that this approach is rather verbose. You always need to add the `db_migration_test` annotation to any migration test we will add in the future. It's very likely that we will forget this at some point. What do you think about the following approach:

* Remove the `db_migration_test` annotation and distinguish the migration tests for the old and the new versioning schema based on the filename. When the filename of the test case matches the versioning schema of the regex, skip it when the timestamp is older than 30 days.
* Enforce the constraint that any database migration test need either the `slowtest` annotation or it's filename needs to match the new versioning schema. That way we protect ourselves against typos in the name of the file.

Sorry, I noticed this problem only now.

@sanderr Do you have an opinion on this?

Reply:
I agree with your suggestion.

Reply:
Do I understand correctly that you would add a mechanism before the run of each test that would look a the path of the file to determine if it is a migration test or not, then use the regex to see what do to/which annotation is needed?

Reply:
Yes

Reply:
This shouldn't be to hard to change :) And I agree it wouuld make life easier. I'll change it